app/controllers/short_movies_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ShortMoviesController < ApplicationController
+  def show
+    @short_movies = ShortMovie.all
+  end
+
+  def create
+    @short_movie = ShortMovie.new(short_movies_require)
+
+    if @short_movie.save
+      puts 'Video saved!'
+    else
+      puts 'Video not saved!'
```

Comment:
I will work on this one later, right now it is just a space holder, but i should comment it out!